rse-publish.Rmd

```diff
@@ -0,0 +1,544 @@
+```
+
+## Objectives {#publish-objectives}
+
+```{r, child="objectives/publish.md"}
+```
+
+## Introduction {#publish-intro}
+
+This lesson looks at what should be included in reports and how best to do that.
+We use the generic term "report" to include research papers,
+summaries for clients,
+and everything else that is shorter than a book and is going to be read by someone else.
+
+Our motivation is summed up in this quotation:
+
+> An article about computational science in a scientific publication is *not* the scholarship itself,
+> it is merely *advertising* of the scholarship.
+> The actual scholarship is the complete software development environment
+> and the complete set of instructions which generated the figures.
+>
+> -- Jonathan Buckheit and David Donoho, paraphrasing Jon Claerbout, in @Buck1995
+
+As the quote suggests,
+modern publishing involves much more than simply producing a written report.
+It involves providing readers with the data underpinning the report,
+as well as any code written in analysing the data.
+
+While the definition of data (and its associated metadata)
+is relatively easy to understand,
+code can come in many different forms.
+Here we distinguish between “analysis code” written
+solely for the purpose of the report (e.g. to produce a figure)
+and “analysis software” that is formally packaged and released for use by a wider audience.
+Of course, in reality the analysis code/software written during the process of preparing
+a report can often lie on a continuum between those two definitions. 
+
+While some written reports, datasets, software packages and/or analysis code
+can't be published without violating personal or commercial confidentiality,
+every researcher's default should be to make all these components
+of their work as widely available as possible.
+That means publishing it in an open access venue (Chapter \@ref(inclusive))
+so that people who aren't in academia can find and access it.
+
+
+## Identification {#publish-identifiers}
+
+Before publishing anything,
+we need to understand the systems used to identify works
+and their authors. 
+
+A [Digital Object Identifier](glossary.html#doi) (DOI)
+is a unique identifier for a particular version of a particular digital artifact
+such as a report, a dataset, or a piece of software.
+DOIs are written as `doi:prefix/suffix`,
+but you will often also see them represented as URLs like `http://dx.doi.org/prefix/suffix`.
+In order to be allowed to issue a DOI,
+online platforms (e.g. academic journals, data archives)
+must guarantee a certain level of security, longevity and access.
+
+An [ORCID](glossary.html#orcid) is an Open Researcher and Contributor ID.
+You can [get an ORCID][orcid] for free,
+and you should include it in publications
+because people's names and affiliations change over time.
+
+
+## Publishing a written document {#publish-written}
+
+The best option for publishing a written report (with a platform that issues a DOI)
```

Comment:
why "written"?

Reply:
Good point. I was trying to distinguish the document with the words in it from all the supplementary stuff (code, software, dataset), but saying "written document" sounds a little silly. I've now removed that language.